openflow_input/bsn_tlv

```diff
@@ -963,3 +963,16 @@ struct of_bsn_tlv_data_mask : of_bsn_tlv {
     uint16_t length;
     of_octets_t value;
 };
+
+enum ofp_bsn_port_usage(wire_type=uint16_t) {
```

Comment:
Optional comment: 
Could alternatively model this as a bitmask enum (see https://github.com/floodlight/loxigen/blob/master/openflow_input/standard-1.5#L1241, with transmit/receive being a bit).

Would result in a slightly cleaner way to ask the question "is receive enabled"?